planar/triangulate/delaunaytriangulationbuilder.go

```diff
@@ -0,0 +1,214 @@
+/*
+Copyright (c) 2016 Vivid Solutions.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v1.0
+and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+and the Eclipse Distribution License is available at
+
+http://www.eclipse.org/org/documents/edl-v10.php.
+*/
+
+package triangulate
+
+import (
+	"sort"
+
+	"github.com/go-spatial/geom"
+	"github.com/go-spatial/geom/cmp"
+	"github.com/go-spatial/geom/planar/triangulate/quadedge"
+)
+
+/*
+A utility class which creates Delaunay Triangulations
+from collections of points and extract the resulting
+triangulation edges or triangles as geometries.
+
+Author Martin Davis
+Ported to Go by Jason R. Surratt
+*/
+type DelaunayTriangulationBuilder struct {
+	siteCoords []quadedge.Vertex
+	tolerance  float64
```

Comment:
Is there  a reason that `tolerance`  can not be public?

Seems that the Java implementation allowed the user to set this as needed.

Reply:
We could expose a SetTolerance method or similar which would then also update the tolerance of the QuadEdgeSubdivision. This is doable, but I'd need to go through and look at all the implications in the QuadEdgeSubdivision. It is likely that setting the tolerance after create is called will necessitate recreating the subdivision. All doable, but unless you have a specific use case I think keeping it private is the simplest approach.

Reply:
Let's keep it private.